wdio.conf.js

```diff
@@ -21,9 +21,9 @@ exports.config = {
   // then the current working directory is where your `package.json` resides, so `wdio`
   // will be called from there.
   //one file
-  specs: ["./Unit_9/test/test2.spec.js"],
-//folder with tests
- // specs: ["./Unit_9/test/*.js"],
+  specs: ["./test/specs/suite1.spec.js"],
```

Comment:
We'd better to remove example from the folder and use something like ./test/specs/*.spec.js here